settings.php

```diff
@@ -114,6 +114,24 @@
             new lang_string('settings:presignedurl:presignedminfilesize', 'tool_objectfs'),
             new lang_string('settings:presignedurl:presignedminfilesize_help', 'tool_objectfs'), 0, PARAM_INT));
 
+        $options = [
+            'video/mp4' => 'video/mp4',
+            'text/plain' => 'text/plain',
+            'image/png' => 'image/png',
+            'text/html' => 'text/html',
+            'text/css' => 'text/css',
+            'application/pdf' => 'application/pdf'
+        ];
+        $settings->add(
+            new admin_setting_configmulticheckbox(
```

Comment:
Is it possible to use admin_setting_filetypes ?

Reply:
updated